buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -46,19 +46,20 @@
  */
 public abstract class AbstractByteBuf extends ByteBuf {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(AbstractByteBuf.class);
-    private static final String LEGACY_PROP_CHECK_ACCESSIBLE = "io.netty.buffer.bytebuf.checkAccessible";
-    private static final String PROP_CHECK_ACCESSIBLE = "io.netty.buffer.checkAccessible";
+    private static final String LEGACY_PROP_CHECK_ACCESSIBLE = "siren.io.netty.buffer.bytebuf.checkAccessible";
+    private static final String PROP_CHECK_ACCESSIBLE = "siren.io.netty.buffer.checkAccessible";
     static final boolean checkAccessible; // accessed from CompositeByteBuf
-    private static final String PROP_CHECK_BOUNDS = "io.netty.buffer.checkBounds";
+    private static final String PROP_CHECK_BOUNDS = "siren.io.netty.buffer.checkBounds";
```

Comment:
@rendel this `checkBounds` property is new. Should we default it to `false` in our fork, or leave it as it is upstream, i.e., `true` ?

Reply:
discussed with renaud, we make it `false` by default. @GeorgeAp you've got nothing to change, just the `README.md` file to update the test command line.

Reply:
Done https://github.com/sirensolutions/netty/pull/17/commits/c8bc7e80a12416d2a2e5399ed050af110b00ca95